vnext/Microsoft.ReactNative/Views/ReactRootControl.cpp

```diff
@@ -269,6 +270,9 @@ void ReactRootControl::ShowInstanceLoaded(Mso::React::IReactInstance &reactInsta
 }
 
 void ReactRootControl::ShowInstanceError(Mso::React::IReactInstance &reactInstance) noexcept {
+  if (!m_isDevModeEnabled)
```

Comment:
>if (!m_isDevModeEnabled) [](start = 2, length = 24)

Is it intentional to hide RedBox in non-dev mode? I understand about hiding progress, but error reporting could be important.

Reply:
I'm looking more into our error handling.  Looks like there is a huge set of functionality missing currently.

But the redbox shows developer information, it shouldn't show to end users.

Reply:
It can also leak internals that users shouldn't see 🤷‍♀️

Reply:
OK, it makes sense then.